src/sites/link/linkdrop.net.js

```diff
@@ -306,6 +311,36 @@
   }
 
 
+  class InvisibleRecaptchaHandler extends RecaptchaHandler {
+
+    constructor () {
```

Comment:
btw: why does every class have these? It would usually call the parent constructor anyway if there is none in the child class.

Reply:
Yes that's not necessary I guess.
I guess there was something need to pass into the constructor (I cannot even remember lol), but it was eventually removed.